<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Leida</class>
 <widget class="QMainWindow" name="Leida">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leida</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupSet">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>设置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="addrLab">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>地址：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="IpAddrEdit" name="addrInput" native="true">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="portLab">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>端口：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="portInput">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="xLab">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>横向范围：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QSpinBox" name="xInput">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="9">
        <widget class="QLabel" name="yLab">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>纵向范围：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="10">
        <widget class="QSpinBox" name="yInput">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="11">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="12">
        <widget class="QPushButton" name="startBtn">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupView">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>显示</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="viewWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupData">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>数据</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1360</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IpAddrEdit</class>
   <extends>QWidget</extends>
   <header>ipaddredit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>menuBar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>Leida</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
